docs/dev/table/modules.md

```diff
@@ -101,8 +93,6 @@ The following types are supported out of the box.
 
 {% highlight yaml %}
 modules:
-   - name: core
-     type: core
```

Comment:
if in 1.11 we add back core module, this would be a problem because sql client requires users listing all modules in yaml file. So if users just upgrade to 1.11 without putting core module here, sql client would assume users are explicitly not loading core.

I'm not sure what a better solution can be though, thus I'm neutral to this change

Reply:
This is a valid concern. Maybe we should not remove the `CoreModule` after all. I'm wondering if it would be hard to fix it instead.